packages/pluggableWidgets/rich-text-web/src/components/MainEditor.tsx

```diff
@@ -0,0 +1,6 @@
+import { useCKEditor, CKEditorHookProps } from "ckeditor4-react";
+
+export const MainEditor = ({ config }: { config: CKEditorHookProps<string> }): null => {
+    useCKEditor(config);
```

Comment:
Cant we just use this hook instead of creating a new component without anything else relevant?

Reply:
nope, https://github.com/mendix/widgets-resources/pull/1125#discussion_r772403719
We already discussed a possible solution with @iobuhov and the only way to have ckeditor working is this.